src/stores/modules/Groups.store.js

```diff
@@ -31,8 +31,8 @@ const actions = {
 
   async ADD_GROUP({commit, state}, groupname) {
     commit('fetching');
-    var result = await api.groups.addNewGroup(groupname);
-    if(result.errors) {
+    const {success} = await api.groups.addNewGroup(groupname);
```

Comment:
This does change the semantics as it now depends on the status code as opposed to the contents of the response body, but they should match 100% of the time and this is more correct

Reply:
This seems fine to me.